samples/net/sockets/tls_configurations/src/main.c

```diff
@@ -0,0 +1,231 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(tls_configuration_sample, LOG_LEVEL_INF);
+
+#include <zephyr/kernel.h>
+#include <errno.h>
+#include <stdio.h>
+
+#include <zephyr/posix/sys/eventfd.h>
+
+#include <zephyr/net/socket.h>
+#include <zephyr/net/tls_credentials.h>
+#include <zephyr/net/net_if.h>
+#include <zephyr/sys/util.h>
+
+#define CA_CERTIFICATE_TAG 1
+#define PSK_TAG 2
```

Comment:
Do you actually need a different tag for certificate and PSK? They're not both used in a same build?

Reply:
not sure, I need to check what  TLS 1.3's psk_ephemeral and also some TLS 1.2 do. However, would this be a problem if we keep both? Would it be better if I make an `enum` here so that we have "no offset"?

Reply:
No, this was a rather minor remark. `enum` or `#define` doesn't matter to me. I was wondering specifically about the need to use two different values, because I didn't know whether PSK and certificate are both used simultaneously in a same configuration. But they actually are?